docs/user/openstack/README.md

```diff
@@ -60,43 +60,49 @@ In order to run the latest version of the installer in OpenStack, at a bare mini
 
 ### Swift
 
-Swift must be enabled.  The user must have `swiftoperator` permissions and
-  `temp-url` support must be enabled. As an OpenStack admin:
-  * `openstack role add --user <user> --project <project> swiftoperator`
-  * `openstack object store account set --property Temp-URL-Key=superkey`
+Swift must be enabled.  The user must have `swiftoperator` permissions and `temp-url` support must be enabled. As an OpenStack administrator:
+
+```sh
+openstack role add --user <user> --project <project> swiftoperator
+openstack object store account set --property Temp-URL-Key=superkey
+```
 
-**NOTE:** Swift is required as the user-data provided by OpenStack is not big
-enough to store the ignition config files, so they are served by swift instead.
+**NOTE:** Swift is required as the user-data provided by OpenStack is not big enough to store the ignition config files, so they are served by Swift instead.
 
 * You may need to increase the security group related quotas from their default
-  values. For example (as an OpenStack admin) `openstack quota set --secgroups 8 --secgroup-rules 100 <project>`
+  values. For example (as an OpenStack administrator):
+
+```sh
+openstack quota set --secgroups 8 --secgroup-rules 100 <project>`
+```
 
 ### RHCOS Image
 
-If you do not have a Red Hat Core OS image already, or are looking for the latest,
- [click here](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest/).
+If you do not have a Red Hat Core OS image already, or are looking for the latest, [click here](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest/).
 
 The installer requires a proper RHCOS image in the OpenStack cluster or project:
-`openstack image create --container-format=bare --disk-format=qcow2 --file rhcos-${RHCOSVERSION}-openstack.qcow2 rhcos`
-
-**NOTE:** Depending on your OpenStack environment you can upload the RHCOS image
-as `raw` or `qcow2`. See [Disk and container formats for images](https://docs.openstack.org/image-guide/image-formats.html) for more information. The installer looks for an image named rhcos. This could be overridden via the `OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE` environment variable if for instance your cloud operator provides the image under a different name.
 
-* The public network should be created by the OSP admin. Verify the name/ID of the 'External' network:
+```sh
+openstack image create --container-format=bare --disk-format=qcow2 --file rhcos-${RHCOSVERSION}-openstack.qcow2 rhcos
 ```
+
+**NOTE:** Depending on your OpenStack environment you can upload the RHCOS image as `raw` or `qcow2`. See [Disk and container formats for images](https://docs.openstack.org/image-guide/image-formats.html) for more information. The installer looks for an image named `rhcos`. This could be overridden via the `OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE` environment variable if for instance your cloud operator provides the image under a different name.
```

Comment:
This is not ideal..

Reply:
cc @crawford

Reply:
@abhinavdahiya can you clarify what the problem here is?

Reply:
Telling users to use `OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE` is not ideal as that env will show users a warning that override is not supported and should only be done for development.

Reply:
Can we file this as a bug in a bz, if its going to be problematic for our release, we would like to patch it

Reply:
Or would you rather we simply don't give them the option to use the Override

Reply:
@abhinavdahiya what is the desired way to set a different boot image in 4.2? Environment variable (as documented here) or option in the installer as implemented at https://github.com/openshift/installer/pull/2226?

We need to come to a resolution for https://bugzilla.redhat.com/show_bug.cgi?id=1741930.